src/components/BugsTable/index.jsx

```diff
@@ -0,0 +1,262 @@
+import { Component } from 'react';
+import { object } from 'prop-types';
+import { withStyles } from 'material-ui/styles';
+import Table, {
+  TableBody,
+  TableHead,
+  TableRow,
+  TableSortLabel,
+  TablePagination,
+  TableCell,
+} from 'material-ui/Table';
+import Toolbar from 'material-ui/Toolbar';
+import Typography from 'material-ui/Typography';
+import { PROJECTS_PAGE_SIZE } from '../../utils/constants';
+
+const columns = [
+  { numeric: false, label: 'Project' },
+  {
+    numeric: false,
+    label: 'Description (#ID)',
```

Comment:
What do you think about using "Summary" instead of "Description"? This represents a summary of the bug/issue -- not really a description.

Reply:
yeah i was initially think of title, but i think summary might be better.